src/edu/first/module/actuators/SpeedController.java

```diff
@@ -175,5 +175,23 @@ public void disable() {
         public void pidWrite(double output) {
             controller.setSpeed(output);
         }
+
+		@Override
+		public void setInverted(boolean isInverted) {
+			// TODO Auto-generated method stub
```

Comment:
In these new methods, either implement their functionality or throw UnsupportedOperationException